pkg/agent/flowexporter/connections/conntrack_test.go

```diff
@@ -164,27 +167,13 @@ func TestConnTrackOvsAppCtl_DumpFlows(t *testing.T) {
 	assert.Equal(t, len(outputFlow), totalConns, "Number of connections in conntrack table should be equal to outputFlow")
 }
 
-func TestSetupConnTrackParameters(t *testing.T) {
-	err := setupConntrackParameters()
-	if err != nil {
-		t.Skipf("Skipping test as trying to set up conntrack parameters returned an error: %v", err)
-	} else {
-		conntrackAcct, err := sysctl.GetSysctlNet("netfilter/nf_conntrack_acct")
-		require.NoError(t, err, "Cannot read nf_conntrack_acct")
-		assert.Equal(t, 1, conntrackAcct, "net.netfilter.nf_conntrack_acct value should be 1")
-		conntrackTimestamping, err := sysctl.GetSysctlNet("netfilter/nf_conntrack_timestamp")
-		require.NoError(t, err, "Cannot read nf_conntrack_timestamp")
-		assert.Equal(t, 1, conntrackTimestamping, "net.netfilter.nf_conntrack_timestamp value should be 1")
-	}
-}
-
 func TestConnTrackSystem_GetMaxConnections(t *testing.T) {
 	connDumperDPSystem := NewConnTrackSystem(&config.NodeConfig{}, &net.IPNet{})
 	maxConns, err := connDumperDPSystem.GetMaxConnections()
 	assert.NoErrorf(t, err, "GetMaxConnections function returned error: %v", err)
-	expMaxConns, err := sysctl.GetSysctlNet("nf_conntrack_max")
-	require.NoError(t, err, "Cannot read nf_conntrack_max")
-	assert.Equal(t, expMaxConns, maxConns, "The return value of GetMaxConnections function should be equal to nf_conntrack_max")
+	expMaxConns, err := sysctl.GetSysctlNet("netfilter/nf_conntrack_max")
+	require.NoError(t, err, "Cannot read netfilter/nf_conntrack_max")
+	assert.Equal(t, expMaxConns, maxConns, "The return value of GetMaxConnections function should be equal to netfilter/nf_conntrack_max")
```

Comment:
I am confused, you removed the `_linux` suffix and the PR description states "Removed the dependency on Linux platform of conntrack unit tests."

But using `sysctl.GetSysctlNet` makes it Linux-specific since this function only exists for Linux

Reply:
To be clear, the only things I would suggest doing here are:
 * editing the PR description / final commit message to remove that item. What we did with this PR is remove the need to be root to run the unit tests. As discussed in https://github.com/vmware-tanzu/antrea/pull/1321#discussion_r499019732
 * maybe keep the `_linux` suffix in the filename

Let me know if I'm missing something.

In a future PR, you can look into the following:
 - enable running unit tests on Windows in Github actions (natively, not within a docker container)
 - split this file into platform-specific tests and platform-agnostic tests

Reply:
Thanks, I have modifited the description and commit message.